pkg/devconsole/tekton-results/results.go

```diff
@@ -94,21 +94,27 @@ func makeHTTPRequest(url, userToken string) (common.DevConsoleCommonResponse, er
 
 	// Load the CA certificate
 	caCert, err := os.ReadFile(tlsCertPath)
-	if err != nil {
+	if err != nil && k8sMode != "off-cluster" {
 		return common.DevConsoleCommonResponse{}, fmt.Errorf("failed to read CA certificate: %v", err)
 	}
 
 	// Create a CertPool and add the CA certificate
 	caCertPool := x509.NewCertPool()
-	if !caCertPool.AppendCertsFromPEM(caCert) {
+	if !caCertPool.AppendCertsFromPEM(caCert) && k8sMode != "off-cluster" {
 		return common.DevConsoleCommonResponse{}, fmt.Errorf("failed to append CA certificate")
 	}
 
 	serviceTransport := &http.Transport{
 		Proxy: http.ProxyFromEnvironment,
-		TLSClientConfig: &tls.Config{
+	}
+	if k8sMode == "off-cluster" {
+		serviceTransport.TLSClientConfig = &tls.Config{
+			InsecureSkipVerify: true,
```

Comment:
Should this be controlled by the `k8s-mode-off-cluster-skip-verify-tls` command-line flag instead?

Reply:
You mean to say start the bridge with flag?

`./bin/bridge --k8s-mode-off-cluster-skip-verify-tls=true`

I tried this but it doesn't work. Still getting tls error `tls: failed to verify certificate: x509: certificate signed by unknown authority`

Reply:
I'm saying that's the configuration we should look at to determine whether to use `InsecureSkipVerify` here.